src/screens/FeedbackScreen.tsx

```diff
@@ -43,7 +43,7 @@ const FeedbackScreen = (props: { loginStorageKey: string; navigation }) => {
         {Object.keys(userData)
           .filter((key) => key !== 'Feedback')
           .map((key) => {
-            return <DataField itemKey={key} value={userData[key]} />;
+            return <DataField key={key} itemKey={key} value={userData[key]} />;
           })}
```

Comment:
You don't have to change this, but since we want both the key and value, `Object.entries` might be more fit for the job. But really, not a big enough difference to warrant a change 😅 
```TypeScript
Object.entries(userData)
  .filter(([key]) => key !== "Feedback")
  .map(([key, value]) => <DataField key={key} itemKey={key} value={value} />);

```